recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/README.md

```diff
@@ -0,0 +1,56 @@
+# AISHELL-1 Non-Autoregressive ASR with [NAR-BERT-ASR](https://arxiv.org/abs/2104.04805) models.
+
+
+# Introduction
+This folder contains recipes for [NAR-BERT-ASR](https://arxiv.org/abs/2104.04805) with [AISHELL-1](https://www.openslr.org/33/), a 150-hour Chinese ASR dataset.
+
+
+# How to run
+Step 1: Install extra dependencies.
+```
+pip install transformers
```

Comment:
you can remove the extra_dependencies.txt file if you install it directly in that way :-)